src/app/credExplorer/WeightConfig.js

```diff
@@ -175,36 +172,30 @@ class EdgeConfig extends React.Component<{
 }
 
 class NodeConfig extends React.Component<{
-  nodeWeights: NodeWeights,
-  onChange: (NodeWeights) => void,
+  nodeWeights: WeightedNodeType[],
```

Comment:
That this array should be mutable is quite surprising. Is it intentional? (Likewise in state, though less so.)

Reply:
Fixed.